kratos/input_output/stl_io.cpp

```diff
@@ -0,0 +1,145 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:         BSD License
+//                   Kratos default license: kratos/license.txt
+//
+//  Main authors:    Pooyan Dadvand
+//
+
+// System includes
+
+// External includes
+
+// Project includes
+#include "includes/define.h"
+#include "input_output/stl_io.h"
+
+namespace Kratos
+{
+
+/// Constructor with  filenames.
+StlIO::StlIO(std::string const& Filename)
```

Comment:
Please use filesystem::path instead, see the ModelPartIO

Reply:
![](https://media.tenor.com/-gS-ft4nRzkAAAAM/interesting-batman.gif)

Reply:
Done